docs/docs/identity-providers/github.md

```diff
@@ -9,63 +9,81 @@ meta:
 
-3. Go to your settings.
+1. Navigate to **Developer settings ➞ OAuth Apps** and select **New OAuth App**.
 
-![GitHub settings](./img/github/github-user-profile.png)
+  ![GitHub OAuth2 Application creation](./img/github/github-oauth-creation.png)
 
-4. Click the Developers settings and create a new OAuth Application.
+1. Create a new OAuth2 application by filling the form fields above with the following parameters:
 
-![GitHub OAuth2 Application creation](./img/github/github-oauth-creation.png)
+   | Field                       | Description                                                         |
+   | --------------------------- | ------------------------------------------------------------------- |
+   | Application name            | The name of your web app.                                           |
+   | Homepage URL                | The homepage URL of the application to be integrated with Pomerium. |
+   | Authorization callback URL  | `https://${authenticate_service_url}/oauth2/callback`, `authenticate_service_url` from your Pomerium configuration. |
 
-5. Create a new OAuth2 application by filling the form fields above with the following parameters:
 
-Field                       | Description
---------------------------- | --------------------------------------------
-Application name            | The name of your web app
-Homepage URL                | The homepage URL of the application to be integrated with Pomerium
-Authorization callback URL  | `https://${authenticate_service_url}/oauth2/callback`, authenticate_service_url from pomerium configuration
+1. After creating the application, select **Generate a new client secret** and save **Client Secret** along with the **Client ID**.
 
+## Create a Service Account
 
-After the application had been created, you will have access to the credentials, the **Client ID** and **Client Secret**.
+To use `allowed_groups` in a policy, an `idp_service_account` needs to be set in the Pomerium configuration. The Service Account for GitHub should be a personal access token with `read:org` permissions
 
-## Service Account
-To use `allowed_groups` in a policy an `idp_service_account` needs to be set in the Pomerium configuration. The Service Account for GitHub should be a personal access token with `read:org` permissions, which can be created at [github.com/settings/tokens/new](https://github.com/settings/tokens/new).
+1. Create a new token at [github.com/settings/tokens/new](https://github.com/settings/tokens/new).
 
-![Personal Access Token](./img/github/github-personal-access-token.png)
+   ![Personal Access Token](./img/github/github-personal-access-token.png)
 
-The format of the `idp_service_account` for GitHub is a base64-encoded JSON document:
+1. The format of the `idp_service_account` for GitHub is a base64-encoded JSON document:
 
-```json
-{
-  "username": "YOUR_GITHUB_USERNAME",
-  "personal_access_token": "GENERATED_GITHUB_ACCESS_TOKEN"
-}
-```
+   ```json
```

Comment:
```suggestion
   ```json tmp.json
```

maybe we should make the connection between this file, and the thing we are base64'ing explicit here (and in the other docs?)

Reply:
It doesn't appear that adding this changes anything in the render.

Reply:
Ah, okay. I guess vuepress doesn't support file name rendering.